### ISSUE TITLE
Comment on logits-scaling BiC layer commited 05042025 3.58UCT+8

### ISSUE THREAD

**Issue by Brynniiee** [Brynniiee/BIC#3]
This is actually not that close to IL2M as discribed in the commit information:
1. IL2M do not use distillation, the author shows its draw back and this also happens in my dataset
2. IL2M do not use ML to train logit scalers, instead it adjusts it based on logit means 

So this commit could be seen as a enhanced BiC layer only.
